gravitee-apim-console-webui/src/management/api/documentation-v4/api-documentation-v4.component.spec.ts

```diff
@@ -115,6 +115,33 @@ describe('ApiDocumentationV4', () => {
     jest.resetAllMocks();
   });
 
+  describe('ngOnInit - homepage filter', () => {
+    it('should filter out pages with homepage=true', async () => {
+      const pages = [
+        fakeMarkdown({ id: 'page-1', name: 'Page 1', homepage: true }),
+        fakeMarkdown({ id: 'page-2', name: 'Page 2', homepage: false }),
+      ];
+      await init(pages, []);
+
+      const component = fixture.componentInstance;
+      expect(component.pages.length).toBe(1);
+      expect(component.pages[0].id).toBe('page-2');
```

Comment:
In an ideal world, you would test the UI rendering instead of the component. `pages` isn't necessarily rendered.